src/gluonts/model/lstnet/_estimator.py

```diff
@@ -66,6 +63,13 @@ class LSTNetEstimator(GluonEstimator):
         Number of channels for first layer Conv1D
     kernel_size
         Kernel size for first layer Conv1D (default: 6)
+    prediction_length
```

Comment:
Note it's inevitable to change the argument location bc of the needed default `None` values for `prediction_length` and `horizon`.